docs/apache-airflow/installation/index.rst

```diff
@@ -316,3 +327,62 @@ Follow the  `Ecosystem <https://airflow.apache.org/ecosystem/>`__ page to find a
+    with more (or less) pods and containers that are needed that might depend on particular choice of
+    the technology/cloud/integration of monitoring etc. etc.
+  * Technical details of database, hardware, network, etc. that your deployment is running on
+  * The complexity of the code you add to your DAGS, configuration, plugins, settings etc. (note, that
+    Airflow runs the code that DAG author and Deployment Manager provide)
+  * The number and choice of providers you install and use (Airflow has more than 80 providers) that can
+    be installed by choice of the Deployment Manager and using them might require more resources.
+  * The choice of parameters that you use when tuning Airflow. Airflow has many configuration parameters
+    that can fine-tuned to your needs
+  * The number of DagRuns and tasks instances you run with parallel instances of each in consideration
+  * How complex are the tasks you run
+
+The above "DAG" characteristics will change over time and even will change depending on the time of the day
+or week, so you have to be prepared to continuously monitor the system and adjust the parameters to make
+it works smoothly.
+
+While we can provide some specific minimum requirements for some development "quick start" - such as
+in case of our :ref:`running-airflow-in-docker` quick-start guide, it is not possible to provide any minimum
+requirements for production systems.
+
+The best way to think of resource allocation for Airflow instance is to think of it in terms of process
+control theory - where there are two types of systems:
+
+1. Fully predictable, with few knobs and variables, where you can reliably set the values for the
+   knobs and have an easy way to determine the behaviour of the system
+
+2. Complex systems with multiple variables, that are hard to predict and where you need to monitor
+   the system and adjust the knobs continuously to make sure the system is running smoothly.
+
+Airflow (and generally any modern system running usually on cloud services, with multiple layers responsible
+for resources as well multiple parameters to control their behaviour) is a complex system and they fall
+much more in the second category. If you decide to run Airflow in production on your own, you should be
+prepared for the monitor/observe/adjust feedback loop to make sure the system is running smoothly.
+
+Having a good monitoring system that will allow you to monitor the system and adjust the parameters
+is a must to put that in practice.
+
+There are few guidelines that you can use for optimizing your resource usage as well. The
+:ref:`fine-tuning-scheduler` is a good starting point to fine-tune your scheduler, you can also follow
+the :ref:`best_practice` guide to make sure you are using Airflow in the most efficient way.
+
+Also, one of the important things that Manages Services for Airflow provide is that they made a lot
```

Comment:
```suggestion
Also, one of the important things that Managed Services for Airflow provide is that they make a lot
```
Also thinking if it could be `managed services` instead of title cased `Managed Services` across all the occurrences in this file 🤔